iocore/net/quic/QUICStream.cc

```diff
@@ -248,9 +248,25 @@ QUICStream::_signal_write_event(bool direct)
   }
 }
 
-void
+QUICError
 QUICStream::_write_to_read_vio(const std::shared_ptr<const QUICStreamFrame> &frame)
 {
+  if (frame->has_fin_flag()) {
+    _closed = true;
```

Comment:
I'm not a big fun of this flag. Can we use QUICStreamState insted?

Reply:
Updated later!